config/ember-try.js

```diff
@@ -12,27 +12,23 @@ module.exports = function() {
       useYarn:   true,
       scenarios: [
         {
-          name: 'ember-lts-2.16',
+          name: 'ember-lts-2.18',
           env:  {
             EMBER_OPTIONAL_FEATURES: JSON.stringify({ 'jquery-integration': true })
           },
           npm: {
             devDependencies: {
-              '@ember/jquery': '^0.6.0',
-              'ember-source':  '~2.16.0'
+              '@ember/jquery': '^0.5.1',
```

Comment:
Downgrade?

Reply:
Probably came from the blueprint, but this also matches the latest minor version, so it doesn't change anything